src/components/Book.js

```diff
@@ -2,40 +2,48 @@ import '../styles/Book.css';
 import { CircularProgressbar } from 'react-circular-progressbar';
 import 'react-circular-progressbar/dist/styles.css';
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
```

Comment:
- [ ] Kindly dispatch the `chechStatus` action for the `Category` component with the string `Under construction` when the user clicks on the `Check status` button.